21.7 fb protocol title length

jonesgaohsu opened the pull request:
#### Rationale
Revision of misnamed feature branch previously submitted 
This FB changes the setScale value for the Protocol Title to 200 to match NIH TItle Length

#### Related Pull Requests
* <!-- list of links to related pull requests (replace this comment) -->

#### Changes
Update of ONPRC_EHRCustomeriser.java line 1555

jonesgaohsu committed:
Update of ONPRC_EHRCustomizer to allow 200 characters for Title of Protocol

jonesgaohsu committed:
Merge remote-tracking branch 'origin/release21.7-SNAPSHOT' into release21.7-SNAPSHOT

jonesgaohsu committed:
Revision2 of Protocol Title Feature Branch

jonesgaohsu commented:
Removed bad naming and created new version to correct the naming error

labkey-jeckels commented on onprc_ehr/src/org/labkey/onprc_ehr/table/ONPRC_EHRCustomizer.java:
```diff
@@ -1545,13 +1545,14 @@ private void customizeRequests(AbstractTableInfo ti)
-        ti.getMutableColumn("title").setScale(80);
-        ti.getMutableColumn("investigatorId").setHidden(false);
+        ti.getMutableColumn("title").setScale(200);
+        Update of ONPRC_EHRCustomizer to allow 200 characters for Title of Protocol        ti.getMutableColumn("investigatorId").setHidden(false);
```
I'm sure this is intended to be a comment but that's now how it's formatted and it's breaking the build. Please correct.

jonesgaohsu replied:
CHanges made and submitted to Lakshmi for review - Thanks

labkey-jeckels reviewed (changes requested):
See my comment about the mangled comment line.

Also, please follow the agreed-upon process of having another ONPRC developer review prior to looping me in. Thanks!

jonesgaohsu committed:
Revision2 of Protocol Title Feature Branch - reset as there was unintended code in line 1555

jonesgaohsu committed:
Update ONPRC_EHRCustomizer.java

Corrected the issue on Line 1555 Yes it was an intended comment that went in the wrong place

jonesgaohsu reviewed (commented):
Changes made

jonesgaohsu commented:
Josh, For some reason I cannot add Raymond as a Reviewer, How can he be added?

labkey-jeckels commented:
@jonesgaohsu Keith and I can add him without problems. Are you using his GitHub user name, ohsudev? He hasn't attached his real name to the account so you need to use the user name directly.

jonesgaohsu commented:
With Keith's help got it added for Raymond to review
